test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,161 @@
+                expect(error.statusCode).to.equal(403);
+                expect(error.message).to.equal(OOO_STATUS_ALREADY_EXIST);
+            }
+        });
+
+        it("should return undefined when all validation checks passes", async function() {
+            const response = await validateUserStatus(
+                testUserId,
+                true,
+                testUserStatus
+            );
+            expect(response).to.not.exist;
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)